### DISCUSSION TITLE
Creation of events (Christmas, Thanksgiving, etc)

### DISCUSSION THREAD

**Discussion by gabt92** [mealie-recipes/mealie#811] [Feature Request]
**Before submitting this feature request I have**
- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
There is only the notion of days in the planner, but it would be awesome if we were able to create personalized events.

For example, create an event for Christmas Eve or for a birthday and prepare everything for it. 

Some ideas:
- Next year, the meal planner could propose recipes based on previous events that occurred at the same date.
- Add events to the sidebar momently until it happens.
- Having pre-determined events (that can be created in the admin) like Christmas, Halloween, Birthdays so that if I'm looking for "Christmas", I would be able to retrieve the previous meals planned for those events.
- Create a event with recipes for a group, and assign recipes to different users
- Create statistics based on events if the https://github.com/hay-kot/mealie/discussions/809 is accepted. For example, "Most popular recipe for Christmas" and things like that.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature
> I'd be willing to help conceptualize the idea and test it.

**Comment by hay-kot** [comment]
Love this idea. Not sure if I'll have time to get it in before Christmas but here's how I think the data would be Represented

- Top Level Event
    - Event Section 1 (Main, Sides, Breakfast....Whatever)
         - Recipes / Notes
         - Recipes / Notes
    - Event Section 2 (Main, Sides, Breakfast....Whatever)
         - Recipes / Notes
         - Recipes / Notes
    - Event Section 2 (Main, Sides, Breakfast....Whatever)
         - Recipes / Notes
         - Recipes / Notes

Probably would need to have an active/inactive attribute so only active one's are shown by default.

**Comment by hay-kot** [comment]
Created a task for the v1 release for this

https://github.com/hay-kot/mealie/issues/863

**Comment by morenicano** [comment]
Does this let you share the event so that others can add their own dish to it and state what they're contributing to the menu? I'm looking for a solution to organize holiday dinners this year and hoping this can do that.
